src/test/java/core/basesyntax/strategy/OperationStrategyTest.java

```diff
@@ -0,0 +1,79 @@
+package core.basesyntax.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class OperationStrategyTest {
+    private static OperationStrategy operationStrategy;
+    private static Map<String, OperationHandler> map;
+
+    @BeforeClass
+    public static void beforeClass() {
+        map = new HashMap<>();
+        map.put("b", new BalanceOperationHandler());
+        map.put("s", new SupplyOperationHandler());
+        map.put("p", new PurchaseOperationHandler());
+        map.put("r", new ReturnOperationHandler());
+        operationStrategy = new OperationStrategy(map);
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void getByOperation_emptyOperation_NotOk() {
+        Class<?> expected = BalanceOperationHandler.class;
+        Class<?> actual = operationStrategy.getByOperation("").getClass();
+        assertEquals("Valid method reaction, no such operation with input parameter ",
+                expected, actual);
```

Comment:
```suggestion
        Class<?> actual = operationStrategy.getByOperation("").getClass();
```
also you could use assertNull()